### PR TITLE
feat: Revamp profile settings screen

### PR DESCRIPTION
**Pull request by muhsin-k**
- Renamed `personal` route with `profile` route.
- Removed all the unused codes related to old profile settings page.

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/settings/profile/ChangePassword.vue [hunk 1]
```diff
@@ -1,82 +1,73 @@
 <template>
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/profile/ChangePassword.vue:1]: @muhsin-k Can't we move this component to composition API?

**Review by iamsivin — COMMENTED** [review]
@muhsin-k Added a small comment. The rest looks good. And let us remove this header from the profile page.
image

**Review by scmmishra — APPROVED** [review]

File: app/javascript/dashboard/routes/dashboard/settings/profile/Wrapper.vue [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<template>
+  <div
+    class="flex flex-col justify-between flex-1 h-full m-0 overflow-auto bg-white dark:bg-ash-900"
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/profile/Wrapper.vue:3]: @muhsin-k `dark:bg-ash-900` not working, let us use `bg-slate-900`
```suggestion
    class="flex flex-col justify-between flex-1 h-full m-0 overflow-auto bg-white dark:bg-slate-900"
```

**Review by iamsivin — APPROVED** [review]
Looks good

**Pull request merged**